assets/helpers/visitToken/reducer.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+// ----- Imports ----- //
+
+import type { Action } from './actions';
+
+
+// ----- Types ----- //
+
+export type VisitToken = {
+    token: ?string,
+};
```

Comment:
Does the state here need to be an object? Could it be:

```jsx
type VisitToken = ?string;
```

Reply:
fixed in next pr